website/content/docs/commands/var/list.mdx

```diff
@@ -0,0 +1,137 @@
+---
+layout: docs
+page_title: "Command: var list"
+description: |-
+  The "var list" command lists data from Nomad's variables datastore at
+  or beginning with the given prefix.
+---
+
+# Command: var list
+
+The `var list` command returns a list of variable paths accessible to the
+current user, optionally filtered to paths containing a provided prefix. Do not
+encode sensitive information in variable paths. The variable's items are not
+accessible via this command.
+
+When using pagination, the next page token is provided as part of the output of
+the command. When the output format is JSON, the returned variable list is
+wrapped with additional metadata that contains the next page token. For non-JSON
+output, the next page token is printed as a message to standard error and
+standard output contains the variables from the specifically requested page.
+
+
+## Usage
+
+```plaintext
+nomad var list [options] [<prefix>]
```

Comment:
Why brackets and alligators?

Reply:
Optional argument with a value. A fairly standardized style (used in parsers like [docopt](http://docopt.org/)), where `[options]` is a reference to the list of options below but `[<prefix>]` is an optional argument (vs `<prefix>` for a required argument)